src/database.ts

```diff
@@ -1307,6 +1354,58 @@ class Database extends GrpcServiceObject {
       callback!(err, transaction);
     });
   }
+
+  /**
+   * Query object for listing database operations.
+   *
+   * @typedef {object} GetDatabaseOperationsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetDatabaseOperationsResponse
+   * @property {IOperation[]} 0 Array of {@link IOperation} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List pending and completed operations for the database.
+   *
+   * @see {@link Instance.getDatabaseOperations}
+   *
+   * @param query query object for listing database operations.
+   * @returns {Promise<GetDatabaseOperationsResponse>} when resolved, contains
+   *     a paged list of database operations.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const database = instance.database('my-database');
+   * const [operations] = await database.getDatabaseOperations();
+   */
+  async getDatabaseOperations(
```

Comment:
Would it make sense to name this "getOperations()" instead, since the user would already know the operations will be related to the database they have referenced?

Reply:
Yeah, I agree. I changed it to `getOperations()`.